src/index.html

```diff
@@ -1,13 +1,49 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card-container">
+      <div class="card-product" data-qa="card">
+        <div class="card-product__image">
+          <img
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27&#34 Retina 5K Monoblock"
+            class="card-product__image_size"
+          >
+        </div>
+
+        <div class="card-product__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+        <div class="card-product__code">Product code: 195434</div>
+
+        <div class="card-product__reviews">
+          <div class="card-product__reviews_stars">
```

Comment:
Please rename this class to _stars_, because logically it's separate bem element